apps/backend/src/services/lib/item.service.js

```diff
@@ -1,231 +1,236 @@
-import { Item } from "../../models/lib/item.model.js";
+import { Item } from '../../models/lib/item.model.js';
 import moment from 'moment-timezone';
 
 const getUserItems = async (me) => {
-    const items = await Item.find({
-        user: me,
-        isCompleted: false,
-        isArchived: false,
-        isDeleted: false,
-        spaces: { $exists: true, $eq: [] },
-        status: { $nin: ["archive", "done"] },
-        dueDate: null
-    })
-        .sort({ createdAt: -1 });
-
-    return items;
-}
+  const items = await Item.find({
+    user: me,
+    isCompleted: false,
+    isArchived: false,
+    isDeleted: false,
+    spaces: { $exists: true, $eq: [] },
+    status: { $nin: ['archive', 'done'] },
+  }).sort({ createdAt: -1 });
+
+  return items;
+};
```

Comment:
No changes are needed in getUserItems.

 In the inbox, we are displaying item that don't have any due dates.

Reply:
Oh, ok. I'll change